tensorflow/lite/micro/examples/magic_wand/zephyr_riscv/src/accelerometer_handler.cc

```diff
@@ -0,0 +1,97 @@
+/* Copyright 2019 The TensorFlow Authors. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+==============================================================================*/
+
+#include "tensorflow/lite/micro/examples/magic_wand/accelerometer_handler.h"
+#include <stdio.h>
+#include <zephyr.h>
+#include <device.h>
+#include <string.h>
+#include <drivers/sensor.h>
+
+#define BUFLEN 300
+int begin_index = 0;
+struct device *sensor = NULL;
+int current_index = 0;
+
+float bufx[BUFLEN] = {0.0f};
+float bufy[BUFLEN] = {0.0f};
+float bufz[BUFLEN] = {0.0f};
+
+bool initial = true;
+
+TfLiteStatus SetupAccelerometer(tflite::ErrorReporter* error_reporter) {
+  sensor = device_get_binding(DT_INST_0_ADI_ADXL345_LABEL);
+  if(sensor == NULL) {
+    error_reporter->Report("Failed to get accelerometer, label: %s\n", DT_INST_0_ADI_ADXL345_LABEL);
```

Comment:
`TF_LITE_REPORT_ERROR should be used instead of error_reporter, so log strings can be removed to save space if needed.`

Reply:
@rthadur done